test/compress/class-properties.js

```diff
@@ -182,11 +182,11 @@ static_means_execution: {
         let x = 0;
         // Does not get inlined as it contains an immediate side effect
         class WithStaticProps {
-            static prop = (x = 0 === x ? 1 : "FAIL")
```

Comment:
In this case, I opted to move the `0` back to the right-hand side. This test case is testing code unrelated to comparisons.